### PR TITLE
Improve README and add CONTRIBUTING

### PR DESCRIPTION
**Pull request by loichuder**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,77 @@
 # jupyterhub_moss: JupyterHub MOdular Slurm Spawner
 
-This package extends [`batchspawner.SlurmSpawner`](https://github.com/jupyterhub/batchspawner) to provide a [JupyterHub](https://jupyterhub.readthedocs.io/) [Slurm](https://slurm.schedmd.com/) Spawner with a spawn page allowing to select Slurm resources to use.
+**jupyterhub_moss** is a Python package that aims to provide a configurable [JupyterHub](https://jupyterhub.readthedocs.io/) spawner for [Slurm](https://slurm.schedmd.com/).
 
-## Example
+**jupyterhub_moss** is a Python package that aims to provide:
+- A [JupyterHub](https://jupyterhub.readthedocs.io/) [Slurm](https://slurm.schedmd.com/) Spawner that can be configured by setting the available partitions. It is an extension of [`batchspawner.SlurmSpawner`](https://github.com/jupyterhub/batchspawner).
+- An associated spawn page that changes according to the set Spawner partitions and allows the user to select Slurm resources to use.
```
Comment (t20100) [inline README.md:7]: Spawner partitions ->  Slurm partitions

**Review by t20100 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (loichuder) [reply README.md:7]: Well, the spawn page changes according to the partitions that were set in the `Spawner` which can be different from the Slurm partitions...

Comment (t20100) [reply README.md:7]: yet they need to correspond to Slurm partitions ;) but that's fine

**Comment by kif** [comment]
LGTM

**Review by t20100 — APPROVED** [review]

**Comment by t20100** [comment]
LGTM

**Pull request merged**